.github/workflows/check.yml

```diff
@@ -0,0 +1,30 @@
+name: Test suite
+
+on: [push, pull_request]
+
+jobs:
+  build:
+    runs-on: ubuntu-latest
+    container:
+      image: registry.gitlab.com/islandoftex/images/texlive:latest
+
+    steps:
+    - uses: actions/checkout@v2
+      with:
+        fetch-depth: 0
+
+    - name: Install PGF
+      run: |
+        tlmgr init-usertree --usertree $PWD
+        echo "TEXMFHOME=$PWD" >> $GITHUB_ENV
```

Comment:
I'm not sure this is actually needed: I didn't drop it in my PR, but that was mainly as I was changing as little as possible. Won't _hurt_, but it's also probably a distraction now.

Reply:
(Same comment applies to other `.yml` files, of course.)

Reply:
Good point, I'll get rid of this.